templates/index.html

```diff
@@ -102,6 +102,12 @@ <h1 id="recent-posts">Recent Posts <a id="allposts" href="{{ SITEURL }}/archives
                 <time pubdate="pubdate" datetime="{{ article.date.isoformat() }}">{{ article.locale_date }}</time>
                 </div>
                 </section>
+                {% if RECENT_ARTICLE_SUMMARY %}
+                <p>{{article.summary}}</p>
+                <footer>
```

Comment:
Proposal 1:

Include footer code.

Reply:
Proposal 2:

Exclude footer code.

Reply:
It would be better if "Continue Reading" is aligned to the right i.e. opposite to summary.

Currently Continue Reading and Next article link both are placed on top of each other. Due to minimalism, there is little space between them. To make each link stand out and to reduce chances of clicks on wrong link, it would be better if we push "Continue Reading" to the right. Just like the publication date.

Reply:
In fact, why do we need "Continue Reading" at all? Article link and Continue reading link both point to the same thing. **I think we can do away with Continue Reading completely.**